tractor/_spawn.py

```diff
@@ -369,6 +397,11 @@ async def new_proc(
                     f"{subactor.uid}")
                 nursery.cancel_scope.cancel()
 
+                # if errors:
```

Comment:
This was the alternative approach mentioned in #287: cancelling the `ria_nursery` instead of raising any final error retrieved from the portal.

I think this approach is a bit more, hard to extend wrt supervisor strats being overloaded by a user.